recipe/meta.yaml

```diff
@@ -26,13 +26,21 @@ requirements:
     - six >=1.10.0
     - wheel
     - werkzeug >=0.11.10
+    - markdown >=2.6.8
+    - bleach ==1.5.0
+    - backports.weakref ==1.0rc1
+    - html5lib ==0.9999999
```

Comment:
Admittedly much of this is from before your PR, but could you please drop everything from `build` that isn't strictly required for building the package. Probably can get away with `python`, `pip` only. Then please try re-rendering. Also please make sure you have the latest `conda-smithy` and `conda-build-all`.